addons/mail_group/models/mail_group.py

```diff
@@ -113,13 +130,29 @@ def _compute_mail_group_message_moderation_count(self):
         }
 
         for group in self:
-            group.mail_group_message_moderation_count = result_per_group.get(group.id)
+            group.mail_group_message_moderation_count = result_per_group.get(group.id, 0)
 
     @api.depends('member_ids')
     def _compute_member_count(self):
         for group in self:
             group.member_count = len(group.member_ids)
 
+    @api.depends_context('uid')
+    def _compute_is_member(self):
+        if not self or self.env.user._is_public():
+            self.is_member = False
+            return
+
+        # SUDO to bypass the ACL rules
+        members = self.env['mail.group.member'].sudo().search([
+            ('partner_id', '=', self.env.user.partner_id .id),
```

Comment:
Is it working ? It seems there is a space between ``partner_id``and ``.id`` :eyes:

Reply:
Hum hum, wait yes it work !
Super strange, I remove this space

![screenshot_spect](https://user-images.githubusercontent.com/54316226/131505610-e8cbe95f-79df-4f3b-aee6-d41d8ae04c6c.png)